packages/components/organisms/f-offers/src/components/NoOffersFound.vue

```diff
@@ -8,6 +8,7 @@
             image-url="https://d30v2pzvrfyzpo.cloudfront.net/a/hw/img/decoration/oi_no-results-image.svg"
             :image-align="imageAlign"
             :text-size="textSize"
+            stack-when-narrow
```

Comment:
No other references found of `stack-when-narrow`

Reply:
correct, I need to update this to the newer media element, but will be doing that as part of next PR.